ssrc_config/config_indoor.txt

```diff
@@ -53,13 +50,3 @@ param set MPC_Z_VEL_MAX_UP 1.0
 # Smoothing trajectory a bit when using AutoLineSmoothVel
 param set MPC_JERK_AUTO 8
 param set MPC_ACC_HOR 3
-
-# Default Flight Modes and Kill Switch
-param set COM_FLTMODE1 0
-param set COM_FLTMODE4 2
-param set COM_FLTMODE6 11
-param set RC_MAP_FLTMODE 5
-param set RC_MAP_KILL_SW 7
-
-# Power supply enablement for cloud for Saluki
-param set CBRK_SUPPLY_CHK 894281
```

Comment:
According to the parameter reference, this disable the power valid checks in the commander. Did we test this on a drone that its not causing issues on both Saluki and Pixhawk5x?

Reply:
Yes, this setting was on only because Saluki had faulty power valid checks. It has been corrected, and this parameter is no longer needed.